module_utils/snapshot/util.py

```diff
@@ -0,0 +1,101 @@
+import errno
```

Comment:
The `module_utils` directory is only used by Ansible modules - https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general.html - and requires a specific header, imports, directives, etc.  Unless you are planning on developing an Ansible module instead of a `script`, it would be better to put this code inline in your script.